site/src/content/docs/commands/zarf_tools_helm.md

```diff
@@ -48,6 +48,9 @@ Subset of the Helm CLI that includes the repo and dependency commands for managi
 
 * [zarf tools](/commands/zarf_tools/)	 - Collection of additional tools to make airgap easier
 * [zarf tools helm dependency](/commands/zarf_tools_helm_dependency/)	 - manage a chart's dependencies
+* [zarf tools helm diff](/commands/zarf_tools_helm_diff/)	 - Preview helm upgrade changes as a diff
```

Comment:
Likely an oddity of the make command - curious why these updates to `zarf tools helm` are included.